pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/stateful/ReadingState.java

```diff
@@ -9,54 +9,47 @@
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 
+/**
+ * Stores and updates last-read timestamp in the persistent state (SQLite database) *
+ */
 public class ReadingState {
-
     private static final Logger log = LoggerFactory.getLogger(ReadingState.class);
     public final Connection connection;
-
-    public ReadingState(Connection connection){
+    public ReadingState(Connection connection) {
         this.connection = connection;
-        log.info("connection: {}", this.connection);            
+        log.info("connection: {}", this.connection);
         try {
             try (final Statement statement = connection.createStatement()) {
-                statement.execute(
-                        "create table if not exists LastReadingState (" +
-                            "id integer primary key check(id=0)," +
-                            "lastTimestamp string)");
+                statement.execute("create table if not exists LastReadingState ("
+                        + "id integer primary key check(id=0)," + "lastTimestamp string)");
```

Comment:
Same autoformatting problem here with `+` characters.

Reply:
Changed the formatting.